contrib/deploy-aap/deploy.sh

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+set -m
+
+# variables
+# #########
+# uncomment it, change it or get it from gh-env vars (default behaviour: get from gh-env)
+# export KUBECONFIG=/root/admin.kubeconfig
+
+if ./verify.sh; then
+
+    # Load common vars
+    source ${WORKDIR}/shared-utils/common.sh
+
+    oc apply -f 01-namespace.yml
+    sleep 2
+
+    echo ">>>> Deploy manifests to install AAP"
+    echo ">>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>"
+    oc apply -f 02-operatorgroup.yml
+    sleep 10
+    oc apply -f 03-subscription.yml -o yaml --dry-run=client | oc apply -f -
+
+    OC_COMMAND=$(oc get csv -n ansible-automation-platform | grep "Ansible Automation Platform" | grep Succeeded | wc -l)
+    timeout=0
+    while [ "${timeout}" -lt "120" ]; do
+        if [[ ${OC_COMMAND} -eq 1 ]]; then
+            ready=true
+            break
+        fi
+        echo "Wait for Ansible Automation Platform"
+        sleep 5
+        timeout=$((timeout + 1))
+        OC_COMMAND=$(oc get csv -n ansible-automation-platform | grep "Ansible Automation Platform" | grep Succeeded | wc -l)
+    done
```

Comment:
In the common.sh  file there is a check_resources which do exactly this part. An example:
```
echo "Installing NFD operator for ${edgecluster}"
        oc --kubeconfig=${EDGE_KUBECONFIG} apply -f manifests/01-nfd-namespace.yaml
        sleep 2
        oc --kubeconfig=${EDGE_KUBECONFIG} apply -f manifests/02-nfd-operator-group.yaml
        sleep 2
        oc --kubeconfig=${EDGE_KUBECONFIG} apply -f manifests/03-nfd-subscription.yaml
        sleep 2

        check_resource "crd" "nodefeaturediscoveries.nfd.openshift.io" "Established" "openshift-nfd" "${EDGE_KUBECONFIG}"
```